packages/hint/docs/user-guide/server-configurations/iis.md

```diff
@@ -0,0 +1,260 @@
+# Basic `web.config` for IIS
+
+The following `web.config` should be a good starting point to pass most of
+`webhint`'s checks that require modifying the server configuration.
+
+There are some assumptions though:
+
+* The site is static. If you are using node with iisnode, asp.net, etc.
+  you will have to add the required configuration (but most of this
+  configuration should still be valid).
+* All the static assets are in the folder `dist/static`.
+* The static resources (CSS, JS, images, etc.) have precompressed `gzip`
```

Comment:
> JavaScript

Reply:
Do you mean `zopfli`?

Reply:
Nope, we just need a gzip file. It could be zopfli or regular gzip as long as it ends up with `.gz`.